src/abi/json-abi-format.md

```diff
@@ -26,51 +32,54 @@ we define the following expressions:
 
 The ABI of a contract is represented as a JSON object containing the following properties:
 
-- `"types`": an array describing all the _type declarations_ used (or transitively used) in the ABI. Each _type declaration_ is a JSON object that contains the following properties:
-  - `"typeId"`: a unique integer ID.
+- `"specVersion"`: a string representing the version pointing to this document versioning. When `specVersion` is incremented, it will usually incur an increment in `abiVersion` too.
```

Comment:
Is `specVersion` replacing the `encoding` property?

![encoding](https://github.com/FuelLabs/fuel-specs/assets/26660/31c91ca3-9d42-4360-a899-0004607a425a)